### PR TITLE
Update PHPCS config

### PR DESCRIPTION
**Pull request by swissspidy**
See https://github.com/wp-cli/wp-cli-tests/issues/175

### CODE REVIEW

**Review by jrfnl — COMMENTED** [review]
Line 136 needs to be updated from `file_system_read` to `file_system_operations`.
Ref: https://github.com/WordPress/WordPress-Coding-Standards/wiki/Upgrade-Guide-to-WordPressCS-3.0.0-for-ruleset-maintainers#excluding-a-group-of-checks

You may also want to remove most of the error code based excludes for `WordPress.WP.AlternativeFunctions` (line 82 - 86) as those are now just duplicate.

**Comment by schlessera** [comment]
@swissspidy I didn't realize you had include filesystem functions in here as well. Either close or merge PR #178 as needed.

File: WP_CLI_CS/ruleset.xml [hunk 1]
```diff
@@ -77,28 +77,6 @@
...
-		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_fwrite" />
-		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_is_writable" />
-		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_is_writeable" />
-		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_mkdir" />
-		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_pfsockopen" />
-		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_readfile" />
-		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_rmdir" />
-		<exclude name="WordPress.WP.AlternativeFunctions.file_system_operations_touch" />
-		<exclude name="WordPress.WP.AlternativeFunctions.rename_rename" />
-		<exclude name="WordPress.WP.AlternativeFunctions.unlink_unlink" />
```
Comment (jrfnl) [inline WP_CLI_CS/ruleset.xml]: If you remove these last two as well, then you need to add the `rename` and `unlink` groups to the list of `exclude`s.

Comment (jrfnl) [reply WP_CLI_CS/ruleset.xml]: The error code is build up of `group_name_function_name`, so you can always tell the group based on the error code ;-)

**Review by danielbachhuber — APPROVED** [review]

**Pull request merged**